backends/guest/common/verify.c

```diff
@@ -112,7 +114,7 @@ static int get_current_esl_data(const uint8_t *esl_file, uint8_t **current_esl_d
 			}
 		}
 	} else
-		return INVALID_FILE;
+		prlog(PR_WARNING, "WARNING: %s file does not have data\n", (char *)esl_file);
```

Comment:
Why are we not returning an error code here? It seems like this can only be reached if we fail to allocate memory for the file, or fail to read from the file (`is_file` checks that it exists and can be opened, which *does* return an error code). In both cases, this should be signaled to the calling function that something went wrong.

Reply:
Additional note: it appears `get_auth_data()` does return `INVALID_FILE` in its similar error `else` case, around ~L147

Reply:
Acknowledged, we should return INVALID_FILE in this case as it will avoid unnecessary operations at a later stage.